frontend/docs/docs/develop/emails.md

```diff
@@ -6,10 +6,12 @@ To view and edit email templates, you can run the React Email development server
 
 ```sh
 cd emails
+```
+```sh
 npm run dev
```

Comment:
We use yarn elsewhere (for frontend) - should we stick with yarn here for consistency?
Also, need an `yarn install` step on first run here.

Reply:
Oh you're right, I think this was a leftover from the react email starter template that I didn't swap out.